content/id/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Dalam bidang robotika dan otomatisasi, _control loop_ adalah
+lingkaran tertutup yang mengatur keadaan suatu sistem.
+
+Berikut adalah salah satu contoh loop kontrol: termostat di sebuah ruangan.
+
+Ketika Anda mengatur suhunya, itu mengisyaratkan ke termostat
```

Comment:
Kamu is preferred over "Anda" because it's more friendly. Same comments to the rest of page
```suggestion
Ketika kamu mengatur suhunya, itu mengisyaratkan ke termostat
```